src/opnsense/www/js/widgets/Metadata/Core.xml

```diff
@@ -255,6 +255,10 @@
             <unconfigured>IPsec is currently disabled. Click to configure IPsec.</unconfigured>
             <notunnels>There are currently no tunnels.</notunnels>
             <notavailable>N/A</notavailable>
+            <bytesin>rx</bytesin>
+            <bytesout>tx</bytesout>
```

Comment:
these are not overly useful for translation and/or generally cryptic. What's the goal here?

Reply:
Just doing the same as here:
https://github.com/opnsense/core/blob/4182f19938bb6edb2df4a80f34afb97c1d018216/src/opnsense/www/js/widgets/Metadata/Core.xml#L271

If you want I can change it to `<rx>rx</rx>`, though I wanted to keep it the same name as in the API for consistency.

Reply:
Well I question the overall value of the string "rx" as it translates to something the translator will either not translate or expand  and then your desire to keep it snappy is gone as it would end up reading "Gesendet", "Empfangen" ;)

Reply:
worse yet: "Empfangsrichtung" "Senderichtung" :D

Reply:
So there's two options.

- Either I call it "Bytes In" and "Bytes Out" inside the translation.
- Or I hardcode it to "rx" and "tx" without translation.

Reply:
I worry about "Keine Kinder verbunden". :D

Reply:
"Either I call it "Bytes In" and "Bytes Out" inside the translation." <-- probably most robust and concise